plugins/inputs/prometheus/metrics_receiver_test.go

```diff
@@ -110,3 +114,20 @@ func Test_metricAppender_Commit(t *testing.T) {
 	}
 	assert.Equal(t, expected, *pmb[0])
 }
+
+func Test_loadConfigFromFile(t *testing.T) {
```

Comment:
This test if taManager.enabled is true, could you also add a test for when taManager.enabled is false and see if we are getting the right behavior?